packages/webapp/src/views/developer/biodiversity-gain-site-number.html

```diff
@@ -13,11 +13,10 @@ <h1 class="govuk-heading-l">
           {{ pageHeading }}
         </h1>
 
-        <p>This is the number which connects your development project to the land you're using as your off-site gain.</p>
-        <p>The landowner should give this to you once they’ve registered the biodiversity gain site, and agreed you can use it.</p>
-        <p class="govuk-hint">For example, BGS-010124001</p>
-
-
+        <p>This is the reference number for the land you're using for off-site gain, which you want to allocate to the development project.</p>
+        <p>The landowner will get this number once they've registered the biodiversity gain site.</p>
+        <p class="govuk-hint">For example, BGS-111 222 333</p>
```

Comment:
Can we switch the example back to `BGS-010124001` please. The other example is the wrong format and misleading!